dbt/models/iasworld/schema/iasworld.pardat.yml

```diff
@@ -167,6 +167,32 @@ sources:
             description: Jurisdictions (Municipalities) used as part of Folio number (BCA)
           - name: nbhd
             description: '{{ doc("shared_column_nbhd_code") }}'
+            tests:
+            - row_values_match_after_join:
+                name: iasworld_pardat_nbhd_matches_legdat_township
+                external_model: source('iasworld', 'legdat')
+                external_column_name: user1
+                column_alias: nbhd_code
+                external_column_alias: township_code
+                additional_select_columns:
+                  - model.taxyr
+                  - model.parid
+                  - model.who
+                  - model.wen
```

Comment:
I had to prefix these columns with `model.` since they exist in both the target model and the joined one. It's kind of messy. Maybe we could build some handling into the macro?

Reply:
**[Suggestion, non-blocking]** Yup, I support updating the `row_values_match_after_join` to make this easier. Here's how we do it in the `res_class_matches_pardat` generic:

https://github.com/ccao-data/data-architecture/blob/e313087555fa3afebd91e16445640947175a1570/dbt/tests/generic/test_res_class_matches_pardat.sql#L50-L78

I'm fine with breaking this out into its own issue to avoid blocking this PR.

Reply:
Moved to a new issue! #405